tests/gentype_tests/typescript-react-example/src/Core.gen.tsx

```diff
@@ -21,6 +21,8 @@ export const someFunWithNullUndefinedArg: unknown = someFunWithNullUndefinedArgT
 
 const CoreJS = require('./Core.res.js');
 
+import type {Date_t as Stdlib_Date_t} from './Stdlib.gen';
```

Comment:
@cristianoc How can I avoid this?

Reply:
@cknitt is this what you had in mind? https://github.com/rescript-lang/rescript/pull/7285/commits/295116a0c441cf47153cf92557667ae1c1c99ceb